testsuite/features/secondary/sle_minion_bootstrap_ssh_key.feature

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2016-2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Be able to bootstrap a Salt minion via the GUI using SSH key
+
+  Scenario: Delete SLES minion system profile before bootstrap test
```

Comment:
maybe "before bootstrap with SSH key test" to make it more obvious what bootstrap test it's about.